app-text/calibre/calibre-3.29.0.recipe

```diff
@@ -0,0 +1,174 @@
+SUMMARY="Ebook management application"
+DESCRIPTION="Calibre is an e-book manager. It can view, convert, edit and \
+catalog e-books in all of the major e-book formats. It can also talk to \
+e-book reader devices. It can go out to the internet and fetch metadata for \
+your books. It can download newspapers and convert them into e-books for \
+convenient reading. It is cross platform, running on Linux, Windows and macOS."
+HOMEPAGE="https://calibre-ebook.com/"
+COPYRIGHT="2008-2017 Kovid Goyal"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/kovidgoyal/calibre/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="0eddc3fef236edefa01d7a576e63168b725f5fc872389250eb937280a5018c23"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	calibre$secondaryArchSuffix = $portVersion
+	cmd:calibre
+	cmd:calibre_complete
```

Comment:
```diff
 PROVIDES="
 	calibre$secondaryArchSuffix = $portVersion
-	cmd:calibre
+	cmd:calibre$commandSuffix
-	cmd:calibre_complete
+	cmd:calibre_complete$commandSuffix
 ...
```
(This is to make sure `calibre_x86` and `calibre` won't conflict on x86_64 hybrid for those users who might have built it from source, or when it will be available.